===================================================================
RCS file: /b/source/CVS/src/sys/conf/files,v
retrieving revision 1.55.2.1
diff -c -C2 -r1.55.2.1 files
*** 1.55.2.1	1994/08/15 16:56:26
--- files	1994/10/20 03:47:57
***************
*** 133,136 ****
--- 133,138 ----
  net/if_loop.c		optional loop device-driver
  net/if_ppp.c		optional ppp device-driver
+ net/bsd-comp.c		optional ppp
+ net/ppp_tty.c		optional ppp
  net/if_sl.c		optional sl device-driver
  net/if_tun.c		optional tun device-driver
